apps/docs/src/components/Header.tsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from "react";
+import { HvContainer, HvTab, HvTabs } from "@hitachivantara/uikit-react-core";
+
+import { Classes } from "./usage/Classes";
+import { Description } from "./usage/Description";
+import { Props } from "./usage/Props";
+
+/**
+ * The `Header` component manages a tab-based layout
+ * and dynamically displays specific sections based on the selected tab.
+ */
+export const Header = () => {
+  // State to track the active tab (default is the first tab).
+  const [tab, setTab] = useState(0);
```

Comment:
I don't see the value of these types of comments. Why do we need them? Comments are just like code, they have to be reviewed and maintained. Code should be self-commenting.